docs/network_new/network_module.md

```diff
@@ -0,0 +1,76 @@
+# Network module
+
+## ZBus
+
+Network module is available on zbus over the following channel
+
+| module | object | version |
+|--------|--------|---------|
+| network|[network](#interface)| 0.0.1|
+
+## Home Directory
+
+network keeps some data in the following locations
+| directory | path|
+|----|---|
+| root| `/var/cache/modules/network`|
+
+## Interface
+
+```go
+//Networker is the interface for the network module
+type Networker interface {
+  ApplyNetResource(Network) (string, error)
+  DeleteNetResource(Network) error
+  Namespace(NetID) (string, error)
+}
+```
+
+## Zero-OS networking
+
+### Some First Explanations
+
+Zero-OS is meant to provide services in the Threefold grid, and with grid, we naturally understand that the nodes (or their hosted services) need to be reachable for external users or for each other. So networking in 0-OS is a big thing, even when you assume that 'the network' is ubiquitous and always there, many things need to happen correctly before having a netWORK.  
+For this, apart from all the other absolutely wonderful services in 0-OS, there is the network daemon. If it doesn't succeed it's bootstrap, nothing else will, and 0-OS will stop there.
+
+So it (the network daemon, that is)
+
+- Configures the Node's initial network configuration, so that the Node can register itself.  For now we assume that the Node is connected to a network (ethernet segment) that provides IP addresses over DHCP, be it IPv4 or IPv6, or that there is a Routing Avertisement (RA) daemon for IPv6 running on that network.  
+Only once it has received an IP Address, most other internal services will be able to start. ([John Gage](https://www.networkcomputing.com/cloud-infrastructure/network-computer-again) from Sun said that `The Network is the Computer`, here that is absolutely true)
+
+- Notifies [zinit](https://github.com/threefoldtech/zinit/blob/master/docs/readme.md) (the services orchestrator in 0-OS) that it can register the dhcp client as a permanent process on the intitially discovered NIC (Network Interface Card) and that zinit can start other processes, one of which takes care of registering the node to the grid. (more elaborate explanation about that in [identity service](../identity/readme.md).
+
+- Listens in on the zbus for new or updated Network Resources (NR) that get sent by the provision daemon and applies them.
+
+[Here some thought dumps from where we started working this out](../../specs/network/Requirements.md)
+
+### Jargon
```

Comment:
Let's convert this paragraph into a proper glossary page that we can reference throughout the all network documentation.